CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Jupytext ChangeLog
 ==================
 
+1.16.0-dev (2023-10-??)
```

Comment:
I recommend using a `<!-- SPHINX-START -->` tag to change the header. This is particularly useful for the `README.md` to filter out the badges. Another cool feature is to use markdown reference-style links to correct the links. It should work with myst-parser

Reply:
Oh interesting. In `index.md` we currently have the badges so I was thinking of keeping them, but sure I will filter out the logo. Yes I've seen about the links and it's very useful to convert the links from the `README.md` where there point to docs, to `index.md` where they are local.